dags/common/enricher.py

```diff
@@ -0,0 +1,113 @@
+import os
+import xml.etree.ElementTree as ET
+from typing import Dict
+
+import backoff
+import requests
+from structlog import get_logger
+
+
+class Enricher(object):
+    def __init__(self) -> None:
+        self.logger = get_logger().bind(class_name=type(self).__name__)
+
+    def _get_schema(self):
+        return os.getenv("REPO_URL", "http://repo.qa.scoap3.org") + "/schemas/hep.json"
```

Comment:
this is a challenging one, because we don't have any restriction/checker on how we add url in the env vars. For example we could add either `http://repo.qa.scoap3.org` or `http://repo.qa.scoap3.org/`. It will be safer to do one of the following:

1. Expect the full schema url (which also allow as to give different schemas i.e. inspire or a newer version
2. Build the url using `furl` module

For this specific case I will recommend option 1. WDYT?

Reply:
I agree with option 1 ! Added 👍